package.json

```
@@ -23,7 +23,8 @@
     "lint:sol:fix": "prettier --log-level warn --ignore-path .gitignore '{contracts,test}/**/*.sol' --write",
     "coverage": "scripts/checks/coverage.sh",
     "test": "hardhat test",
-    "test:inheritance": "scripts/checks/inheritance-ordering.js artifacts/build-info/*"
+    "test:inheritance": "scripts/checks/inheritance-ordering.js artifacts/build-info/*",
+    "test:pragma": "scripts/checks/pragma-consistency.js artifacts/build-info/*"
```

Wouldn't be better if we use the `lib/openzeppelin` script?

```suggestion
    "test:pragma": ".lib/openzeppelin/scripts/checks/pragma-consistency.js artifacts/build-info/*"
```

---

The script does 
```javascipt
const { output: solcOutput } = require(path.resolve(__dirname, '../..', artifact));
```

Your approach would use end up looking in tghe wrong place 

```
Error: Cannot find module '/home/amxx/Work/OZ/openzeppelin-community-contracts/lib/@openzeppelin-contracts/artifacts/build-info/2387dcef554d84c915c13f59cc3d0841.json'
```